vnext/src/Libraries/Network/NativeNetworkingWindows.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
What was the diff between NetworkingWindows and core's Networking again? -- Can we annotate the file to make it clearer?

Reply:
Forgot to add this to the PR description (updating...).
`NetworkingWindows` is an analog to `NetworkingIOS and `NetworkingAndroid`.

It's a JS projection that enables the turbo module.

Reply:
Oh.. Networking is forked for IOS and android? ... lame.
Are we the same as one of them at least?  -- You'll need to update the overrides.json file to provide this information anyway.

Reply:
**VERY** forked.

I based the Windows variant mostly on the iOS one, with minor differences.
If you want more context, you can diff the following pairs of files:
- `RCTNetworking.ios.js` vs `RCTNetworking.windows.js`
- `NativeNetworkingIOS.js` vs `RCTNetworkingWindows.js`

Ultimately, the entry point for HTTP calls start with `XMLHttpRequest.js` which calls the platform-specific `RCTNetworking` implementation.